python/pyplugin_installer/installer.py

```diff
@@ -42,11 +42,11 @@
 )
 from qgis.PyQt.QtNetwork import QNetworkRequest
 
-import qgis
 from qgis.core import Qgis, QgsApplication, QgsMessageLog, QgsNetworkAccessManager, QgsSettings, QgsSettingsTree, QgsNetworkRequestParameters
 from qgis.gui import QgsMessageBar, QgsPasswordLineEdit, QgsHelp
 from qgis.utils import (iface, startPlugin, unloadPlugin, loadPlugin, OverrideCursor,
                         reloadPlugin, updateAvailablePlugins, plugins_metadata_parser, isPluginLoaded)
+from qgis import utils
```

Comment:
Isn't possible to add `home_plugin_path` to `from qgis.utils import (...)` instead of this line ?

Reply:
Ok, done. I've cleaned this up